### PR TITLE
Observation Drawer

### PR DESCRIPTION
**Pull request by pmadria3**
This PR updates the observation table in the drawer. 

Screenshot 2023-05-10 at 3 14 44 PM
CTW-1035

### CODE REVIEW

File: src/components/content/observations/helpers/bubble.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <div className={statusToColor(status)}>
+      {result}-{status}
+    </div>
+  ) : (
+    <div className={statusToColor("")}>{result}</div>
+  );
+
+function statusToColor(status: string) {
+  switch (status.toLowerCase()) {
+    case "high":
```
Comment (bryanjknight) [inline src/components/content/observations/helpers/bubble.tsx:17]: Any chance we can clean this up in the model? I feel this list will grow over time and might be duplicated in other components

File: src/components/content/observations/helpers/columns.tsx [hunk 2]
```diff
@@ -37,65 +36,34 @@ export const diagnosticReportColumns: TableColumn<DiagnosticReportModel>[] = [
 
 export const observationsColumns: TableColumn<ObservationModel>[] = [
   {
-    title: "Observation Info",
+    title: "Component",
```
Comment (bryanjknight) [inline src/components/content/observations/helpers/columns.tsx:39]: Confirming we want the column called component

**Review by bryanjknight — APPROVED** [review]
A few questions but no blockers

File: src/components/content/observations/helpers/columns.tsx (hunk 2 above)
Comment (pmadria3) [reply src/components/content/observations/helpers/columns.tsx:39]: Yep confirming it is in the figma 

Screenshot 2023-05-10 at 8 36 34 AM

File: src/components/content/observations/helpers/bubble.tsx (hunk 1 above)
Comment (pmadria3) [reply src/components/content/observations/helpers/bubble.tsx]: Good point slightly reworked this

**Review by bryanjknight — APPROVED** [review]

**Pull request merged**